### DISCUSSION TITLE
Thảo luận

### DISCUSSION THREAD

**Discussion by LeBinhMinh10a6** [LeBinhMinh10a6/Nh-m-4---10C6-IT#8] [General]
mng có add đc columm hay chỉnh trong project ko ?

**Comment by BaoNguyen30** [comment]
Sao hình như tui kh add đc á

**Comment by aimabietduoc** [comment]
qa đây hã

**Comment by BaoNguyen30** [comment]
Z mà đó giờ nhóm mình toàn nhắn bên issues

**Comment by aimabietduoc** [comment]
@LeBinhMinh10a6 tạo project 5 gì chuâ kìa ông oi

**Comment by LeBinhMinh10a6** [comment]
tạo ròi á

**Comment by LeBinhMinh10a6** [comment]
Giờ phân công sao nè mng

**Comment by aimabietduoc** [comment]
ôkee mọi người soạn bài văn đó rùi tui oánh lại vô word r có gì tui gửi r tụi mình sửa lại cho hoàn chỉnh nha

**Comment by BaoNguyen30** [comment]
Z tạo project 5 mình ghi gì trong đó mấy ông

**Comment by aimabietduoc** [comment]
ai bik bà

**Comment by LeBinhMinh10a6** [comment]
ei mng
giờ tui gửi mấy cái bài giảng vô nhe r làm nhe
k phải mỗi làm văn đâu

**Comment by aimabietduoc** [comment]
ủa

**Comment by aimabietduoc** [comment]
dl này là văn thui hay sao mà

**Comment by LeBinhMinh10a6** [comment]
mình còn làm theo định dạng mấy bài giảng của thầy á

**Comment by LeBinhMinh10a6** [comment]
để tui down về rồi gửi lên cho

**Comment by aimabietduoc** [comment]
à oke kkk tui k có máy tính nên k bik

**Comment by BaoNguyen30** [comment]
Mấy định dạng tk lo

**Reply by aimabietduoc** [reply]
tui bị mù công nghệ bà ơi

**Comment by LeBinhMinh10a6** [comment]
làm chung nhe
call nhau ròi record lại post lên đây á

**Comment by aimabietduoc** [comment]
định khi nào làm z mn

**Comment by aimabietduoc** [comment]
soạn cái văn ii r tui oánh bài cho, rùi mn định dạng nha...

**Comment by PhuongTamNguyen** [comment]
t7 cn gì mình ra ngoài r định dạng chung được hong áa

**Comment by LeBinhMinh10a6** [comment]
nhưng mà nhớ record lại , còn ko thì call nhau định dạng á

**Comment by BaoNguyen30** [comment]
Ra ngoài i chứ sợ call khom hiểu ý nhau

**Comment by PhuongTamNguyen** [comment]
đúm ùiiiiii

**Comment by LeBinhMinh10a6** [comment]
sợ có vài người bận việc á , nếu v thì có gì mình coi giờ rồi mình thống nhất với nhau đi

**Comment by aimabietduoc** [comment]
alo moii người ơi viết đoạn văn xong gửi cho tui nha mai tui toinrg hợp rùi tui sửa luôn

**Comment by LeBinhMinh10a6** [comment]
à tình hình là mình F0 rồi nhe mọi người ơi

**Comment by LeBinhMinh10a6** [comment]
![276953688_384199646897612_8296877417564778026_n](https://user-images.githubusercontent.com/92016762/160340159-962fe669-7d07-4864-a686-03c00955aebc.jpg)

**Comment by LeBinhMinh10a6** [comment]
Mình định dạng xong văn bản ròi nhe mọi người

**Comment by LeBinhMinh10a6** [comment]
[Bài làm thực hành word (chưa có kể kỹ thuật).docx](https://github.com/LeBinhMinh10a6/Nh-m-4---10C6-IT/files/8383414/Bai.lam.th.c.hanh.word.ch.a.co.k.k.thu.t.docx)
